### PR TITLE
Version bump 0.1.2

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

**Comment by ocefpaf** [comment]
@conda-forge-admin, please add noarch: python

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,29 +1,26 @@
...
 
 build:
   number: 0
   script: python -m pip install --no-deps --ignore-installed .
-  skip: True  # [py<36]
 
 requirements:
   build:
-    - python
+    - python >=3.6
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @ericmjl by not using the pre-selector and using the constraint here instead we can make the recipe noarch and build it faster.

**Comment by ericmjl** [comment]
Thanks @ocefpaf! Just so that I can reinforce the idea, we can now restrict Python versions without using the selector syntax? That would be awesome!

**Comment by ocefpaf** [comment]
We could always do that and the pre-selector is still preferred (cleaner and `conda-smithy` know what to do when it sees the selector). What we did there is a "hack" to get recipes going with `noarch`.

**Pull request merged**